python/pyspark/pandas/frame.py

```diff
@@ -491,7 +491,8 @@ class DataFrame(Frame, Generic[T]):
 
     >>> import pandas as pd
     >>> sdf = spark.createDataFrame([("Data", 1), ("Bricks", 2)], ["x", "y"])
-    >>> ps.DataFrame(data=sdf, index=pd.Index([0, 1, 2]))
+    >>> with ps.option_context("compute.ops_on_diff_frames", False):
+    ...     ps.DataFrame(data=sdf, index=pd.Index([0, 1, 2]))
```

Comment:
I think maybe now we can just remove the negative cases from the doctests and keep only UTs??

Reply:
I am not sure, @HyukjinKwon wdyt?